plugins/org.eclipse.reddeer.junit/src/org/eclipse/reddeer/junit/requirement/matcher/RequirementVersionMatcher.java

```diff
@@ -34,22 +34,17 @@
  * @author mlabuda@redhat.com
  *
  */
-public class RequirementVersionMatcher extends AbstractRequirementMatcher {
+public class RequirementVersionMatcher extends RequirementAttributeMatcher {
+	
+	private String matchingValue;
```

Comment:
we have matchingValue in both children of RequirementAttributeMatcher. Do you have a reason to not have it in common ancestor together with attributeName?